### ISSUE TITLE
Anyone train and test on Caltech dataset?

### ISSUE THREAD

**Issue by msha096** [lw396285v/CSP.pytorch#9]
Hi,

Is there anyone train and test on Caltech dataset? My model's MR is around 8-9% however the Keras's version's MR is around 4%. No idea why the MR is not promising. (I did not hybrid the data like Keras, is that the reason??)

Any suggestion?

**Comment by Yorionice1** [comment]
I have trained on caltech, but I can't reproduce the paper's number, my miss rate is 7.4%

**Comment by msha096** [comment]
> I have trained on caltech, but I can't reproduce the paper's number, my miss rate is 7.4%


I have the same situation and I am working to find out the reason why the behavior is disappointing on the Caltech dataset. (PyTorch code has a slightly better result on CityPersons dataset than TF) 

Do you have any idea?  I have tried the data hybrid, which is the method that the author of CSP takes in his TF code. However, the MR is still around 8%.

**Comment by msha096** [comment]
@lw396285v Would you like to have a look at it? I have been working on this for about a month. I still have no progress. I have tried to use the same normalization, data hybrid and the MR is always around 8-9%.

**Comment by Yorionice1** [comment]
If I don't use the training methods in the paper，and let the lr decay to 0.00002 after 50 epochs,  the MR is around 5.5%

**Comment by msha096** [comment]
> If I don't use the training methods in the paper，and let the lr decay to 0.00002 after 50 epochs, the MR is around 5.5%

what is the batchsize? did you change other configurations besides the train image size and the test image size when you switch from CityPersons to Caltech?

**Comment by Yorionice1** [comment]
BS==8 on caltech. image size is the same as in the paper.

**Comment by msha096** [comment]
Can you share your train code please? It’s weird I am using bs=12 but
achieved a worse result. My email is msha096@uottawa.ca
Thanks and regards.

**Comment by msha096** [comment]
> BS==8 on caltech. image size is the same as in the paper.


 Can you share your train code please? It’s weird I am using bs=12 but
achieved a worse result. My email is msha096@uottawa.ca
Thanks and regards.

**Comment by LewX** [comment]
@msha096 Hi, what's your training config?

**Comment by msha096** [comment]
> @msha096 Hi, what's your training config?

the same as Keras's one.

**Comment by LewX** [comment]
> > @msha096 Hi, what's your training config?
> 
> the same as Keras's one.

Can you share your training strategy? thx

**Comment by fyture** [comment]
hi, would you share me with the caltech code, thanks a lot! 408261278@qq.com

**Comment by s775980299** [comment]
> I have trained on caltech, but I can't reproduce the paper's number, my miss rate is 7.4%

请问你是怎么复现的，我这里在编译setup.py文件一种报错，如果可以的话加一下我联系方式775980299，这个问题让我心烦好久了，非常感谢
